extensions/amp-date-countdown/1.0/date-countdown-config.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Get locale strings or undefined.
+ * @param {string} locale
+ * @return {DateCountdownLocaleDef|undefined}
+ */
+export function getLocaleStrings(locale) {
+  return LOCALE_WORD[locale];
+}
+
+/**
+ * Strings representing years, minutes, etc. in various locales
+ * @typedef {Array<string>}
+ */
+let DateCountdownLocaleDef;
```

Comment:
No need for this type def. We just should use `!Array<string>` inline.

Reply:
Updated!